vscode/src/debugger/session.ts

```diff
@@ -137,15 +137,6 @@ export class QscDebugSession extends LoggingDebugSession {
         this.breakpointLocations.set(path, mapped);
       } else {
         log.warn(`compilation failed. ${failureMessage}`);
-        sendTelemetryEvent(
```

Comment:
Doesn't that mean the `UserFlowStatus.Succeeded` event below is now incorrect? Should we be sending the failure information in that event?